version/version.go

```diff
@@ -1,5 +1,30 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+)
+
+var log = logf.Log.WithName("version")
+
+//var needs to be used instead of const for ldflags
 var (
-	Version = "0.0.1"
+	Version            = "0.0.1"
```

Comment:
This sort of confuses me. I think we don't care about the value here, so shouldn't be something like `version-change-me` with a comment that this var will be replaced.

Reply:
I would prefer to leave it blank i.e. `""` with a comment